app/src/main/java/com/gh4a/fragment/ReleaseListFragment.java

```diff
@@ -46,9 +46,7 @@ protected Single<Response<Page<Release>>> loadPage(int page, boolean bypassCache
 
     @Override
     protected RootAdapter<Release, ? extends RecyclerView.ViewHolder> onCreateAdapter() {
-        ReleaseAdapter adapter = new ReleaseAdapter(getActivity());
-        adapter.setOnItemClickListener(this);
-        return adapter;
+        return new ReleaseAdapter(getActivity());
```

Comment:
The call to `setOnItemClickListener` was unnecessary because it is already performed by the caller of `onCreateAdapter`: https://github.com/slapperwan/gh4a/blob/43a0a01ed900a69d898ac83c56bfc44d33f8e2ee/app/src/main/java/com/gh4a/fragment/PagedDataBaseFragment.java#L84-L88